src/components/atoms/Badge/Badge.tsx

```diff
@@ -1,17 +1,24 @@
 import type { MantineColor } from '@mantine/core';
 import { Badge as MantineBadge } from '@mantine/core';
 
-export type BadgeStatus = 'error' | 'success' | 'warning';
-
+export type BadgeStatus = 'COMPLETED' | 'CRITICAL' | 'DRAFT' | 'NORMAL';
 export interface BadgeProps {
-  children: string;
+  children: BadgeStatus;
   status: BadgeStatus;
 }
 
 const statusMap: Record<BadgeStatus, MantineColor> = {
-  error: 'red',
-  warning: 'yellow',
-  success: 'green',
+  CRITICAL: 'red',
+  DRAFT: 'orange',
+  NORMAL: 'orange',
+  COMPLETED: 'teal',
+};
+
+const childrenMap: Record<BadgeStatus, string> = {
+  CRITICAL: 'بحرانی',
+  DRAFT: 'تکمیل نشده',
+  NORMAL: 'عادی',
+  COMPLETED: 'تکمیل شده',
 };
```

Comment:
our badge is a customized badge which is an extension of `MantineBadge` I don't think it should know anything about what CRITICAL or DRAFT is

Reply:
not really